addons/lunch/views/lunch_view.xml

```diff
@@ -9,14 +8,14 @@
         <menuitem name="Configuration" parent="menu_lunch" id="menu_lunch_config" sequence="53" groups="group_lunch_manager"/>
 
          <!--View Search to group/filter by Supplier and time-->
-        <record model="ir.ui.view" id="lunch_order_line_search_view">
-            <field name="name">Search</field>
+        <record model="ir.ui.view" id="view_lunch_order_line_search">
```

Comment:
The convention has been clarified since, and is now <model>_<action>_<detail>, so here: lunch_order_line_view_search (model: lunch.order.line, action: view, detail: search view)